### ISSUE TITLE
[1.18.2] `#sel set` crashes

### ISSUE THREAD

**Issue by 5HT2** [cabaletta/baritone#3379]
## Some information
Operating system: arch btw
Java version: `OpenJDK 17.0.3+3`
Minecraft version: `1.18.2`
Baritone version: `1.8.3`
Other mods (if used): listed in the log

## Exception, error or logs
Please find your `latest.log` or `debug.log` in this folder and attach it to the issue

```
[18:52:43] [Render thread/INFO]: [CHAT] [Baritone] > s f magma_block
[18:52:43] [Render thread/INFO]: [CHAT] [Baritone] Filling now
[18:52:43] [Render thread/ERROR]: Unreported exception thrown!
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_2680.method_26204()" because "<parameter1>" is null
	at baritone.dg.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.cu.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.cs.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.ja.b(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.ja.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.kw.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.ek.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at baritone.hd.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) ~[?:?]
	at baritone.hd.a(Unknown Source) ~[baritone-standalone-fabric-1.8.3.jar:?]
	at net.minecraft.class_310.handler$zzj000$runTick(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1730) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1091) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:737) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[18:52:44] [Render thread/INFO]: Minimap session finalized.
[18:52:44] [Render thread/INFO]: Finalizing world map session...
[18:52:44] [Thread-3/INFO]: World map force-cleaned!
[18:52:44] [Render thread/INFO]: World map session finalized.
[18:52:44] [Render thread/INFO]: Stopping worker threads
---- Minecraft Crash Report ----
// Sorry :(

Time: 4/5/22, 6:52 PM
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "net.minecraft.class_2680.method_26204()" because "<parameter1>" is null
	at baritone.dg.a(Unknown Source)
	at baritone.cu.a(Unknown Source)
	at baritone.cs.a(Unknown Source)
	at baritone.ja.b(Unknown Source)
	at baritone.ja.a(Unknown Source)
	at baritone.kw.a(Unknown Source)
	at baritone.ek.a(Unknown Source)
	at baritone.hd.a(Unknown Source)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
	at baritone.hd.a(Unknown Source)
	at net.minecraft.class_310.handler$zzj000$runTick(class_310.java)
	at net.minecraft.class_310.method_1574(class_310.java:1730)
	at net.minecraft.class_310.method_1523(class_310.java:1091)
	at net.minecraft.class_310.method_1514(class_310.java:737)
	at net.minecraft.client.main.Main.main(Main.java:236)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at baritone.dg.a(Unknown Source)
	at baritone.cu.a(Unknown Source)
	at baritone.cs.a(Unknown Source)
	at baritone.ja.b(Unknown Source)
	at baritone.ja.a(Unknown Source)
	at baritone.kw.a(Unknown Source)
	at baritone.ek.a(Unknown Source)
	at baritone.hd.a(Unknown Source)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
	at baritone.hd.a(Unknown Source)
	at net.minecraft.class_310.handler$zzj000$runTick(class_310.java)

-- Affected level --
Details:
	All players: 1 total; [class_746['null'/62096, l='ClientLevel', x=378.62, y=199.00, z=190.31]]
	Chunk stats: 1024, 466
	Level dimension: minecraft:the_nether
	Level spawn location: World: (-496,65,86), Section: (at 0,1,6 in -31,4,5; chunk contains blocks -496,0,80 to -481,255,95), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 63888343 game time, 38529198 day time
	Server brand: Spigot
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:407)
	at net.minecraft.class_310.method_1587(class_310.java:2376)
	at net.minecraft.class_310.method_1514(class_310.java:761)
	at net.minecraft.client.main.Main.main(Main.java:236)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods

-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Linux (amd64) version 5.16.14-arch1-1
	Java Version: 17.0.3, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 463194760 bytes (441 MiB) / 1847590912 bytes (1762 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 5600G with Radeon Graphics
	Identifier: AuthenticAMD Family 25 Model 80 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: Cezanne
	Graphics card #0 vendor: Advanced Micro Devices, Inc. [AMD/ATI] (0x1002)
	Graphics card #0 VRAM (MB): 258.00
	Graphics card #0 deviceId: 0x1638
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 7726.70
	Virtual memory used (MB): 11488.59
	Swap memory total (MB): 0.00
	Swap memory used (MB): 0.00
	JVM Flags: 2 total; -Xms512m -Xmx2048m
	Fabric Mods: 
		autoclicker-fabric: Auto Clicker 1.18-1.3.0-build.16
		autofish: Autofish 0.9.4
		baritone: Baritone 1.8.3
		clientarguments: clientarguments 1.3.0
		clientcommands: Client Commands 2.6.5
		clientcommands-scripting: Clientcommands Scripting 1.3.2
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-config: Cloth Config v6 6.1.48
		continuity: Continuity 1.1.0+1.18.2
		fabric: Fabric API 0.48.0+1.18.2
		fabric-api-base: Fabric API Base 0.4.4+d7c144a860
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.7+d7c144a860
		fabric-biome-api-v1: Fabric Biome API (v1) 8.0.3+801ec85b60
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.12+3ac43d9560
		fabric-command-api-v1: Fabric Command API (v1) 1.1.9+d7c144a860
		fabric-commands-v0: Fabric Commands (v0) 0.2.8+b4f4f6cd60
		fabric-containers-v0: Fabric Containers (v0) 0.1.21+d7c144a860
		fabric-content-registries-v0: Fabric Content Registries (v0) 3.0.2+d82b939260
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.11+3ac43d9560
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.17+2540745460
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.8+d7c144a860
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.19+d7c144a860
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.13+d7c144a860
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.13+d7c144a860
		fabric-item-api-v1: Fabric Item API (v1) 1.3.3+691a79b560
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.10+3ac43d9560
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.12+54e5b2ec60
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.10+b4f4f6cd60
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.0.2+2540745460
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.11+d7c144a860
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.0.4+d1027f7d60
		fabric-models-v0: Fabric Models (v0) 0.3.6+d7c144a860
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.21+d882b91560
		fabric-networking-v0: Fabric Networking (v0) 0.3.8+b4f4f6cd60
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 2.0.3+40c91ec860
		fabric-particles-v1: Fabric Particles (v1) 0.2.12+526dc1ac60
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.8+0d9ab37260
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.13+d882b91560
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.17+d8c7b9ae60
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.11+b4f4f6cd60
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.7+d7c144a860
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 2.0.2+54e5b2ec60
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.13+b4f4f6cd60
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.7+54e5b2ec60
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.2+d1027f7d60
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.18+2de5574560
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.10+d882b91560
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.14+40c91ec860
		fabric-textures-v0: Fabric Textures (v0) 1.0.12+3ac43d9560
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.14+cca23f9360
		fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.0.0+40c91ec860
		fabricloader: Fabric Loader 0.13.3
		ferritecore: FerriteCore 4.2.0
		java: OpenJDK 64-Bit Server VM 17
		lambdynlights: LambDynamicLights 2.1.0+1.17
		lithium: Lithium 0.7.9-SNAPSHOT
		minecraft: Minecraft 1.18.2
		modmenu: Mod Menu 3.1.0
		multiconnect-api: MultiConnect API 1.5.10
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_joml_joml: joml 1.10.2
		phosphor: Phosphor 0.8.1
		pride: Pride Lib 1.1.0+1.17
		sodium: Sodium 0.4.1+rev.e6288d6
		spruceui: SpruceUI 3.3.0+1.17
		xaerominimap: Xaero's Minimap 22.4.0
		xaeroworldmap: Xaero's World Map 1.20.5
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: AMD RENOIR (LLVM 13.0.1, DRM 3.44, 5.16.14-arch1-1) GL version 4.6 (Core Profile) Mesa 22.0.1, AMD
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: Fabric Mods
	Current Language: English (US)
	CPU: 12x AMD Ryzen 5 5600G with Radeon Graphics
#@!@# Game crashed! Crash report saved to: #@!@# /home/f/.local/share/multimc/instances/1.18.1/.minecraft/crash-reports/crash-2022-04-05_18.52.44-client.txt
AL lib: (EE) alc_cleanup: 1 device not closed
Process exited with code 255.
```

## How to reproduce
Add your steps to reproduce the issue/bug experienced here.

1. make selection
2. `#s f any_block`

## Modified settings
To get the modified settings run `#modified` in game

![image](https://user-images.githubusercontent.com/17222512/161864658-777acac0-c927-4df3-af10-11a20d3333de.png)

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [ ] OwO

**Comment by scorbett123** [comment]
Do you by any chance have a button, lever or similar in your inventory. If so remove them.

If you don't have any of these type of items in your inventory, let me know and I will reopen this issue.

**Comment by scorbett123** [comment]
This issue has also been fixed in versions going forwards from #3308.

**Comment by 5HT2** [comment]
> Do you by any chance have a button, lever or similar in your inventory. If so remove them.

Probably lol. Thanks
